plugins/ufm_consumer_plugin/build/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="atabachnik@nvidia.com"
+
+ARG PLUGIN_NAME="ufm_consumer"
+ARG UFM_TAR_PREFIX="ufm"
+ARG UFM_TAR_SUFIX="ubuntu20.x86_64.mofed5"
+ARG DEBIAN_FRONTEND=noninteractive
+ARG BASE_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}
+ARG SRC_BASE_DIR=${PLUGIN_NAME}_plugin
+ARG LOG_LINK=/log
+ARG LOG_DIR=/opt/ufm/files/log
+
```

Comment:
to use bash in build : 
add:
SHELL ['/bin/bash', '-c']

Reply:
changed